### DISCUSSION TITLE
Is there a way to refresh the dashboard

### DISCUSSION THREAD

**Discussion by bertradio** [folke/snacks.nvim#375] [General]
If I change the cwd or open files and then go back to the dashboard the MRU is not updated nor is the list of recent files. Is there a way to have dashboard refresh based on the new situation? I tried Snacks.dashboard.update but that doesn't seem to do anything.

**Comment by kahilah** [comment]
Stumbled across this problem myself and found this thread: https://github.com/folke/snacks.nvim/discussions/128

Trying to get some sort of similar setup working myself atm.
